docs/conversion/sakai_21_0-21_1_oracle_conversion.sql

```diff
@@ -1,11 +1,11 @@
 -- START SAK-45137
 UPDATE rbc_rating SET title = 'Title' WHERE title IS NULL;
 ALTER TABLE rbc_rating MODIFY title VARCHAR2(255) NOT NULL;
-ALTER TABLE rbc_rating ALTER title DROP DEFAULT;
+ALTER TABLE rbc_rating ALTER title DEFAULT NULL;
 
 UPDATE rbc_rating SET points = 0 WHERE points IS NULL;
 ALTER TABLE rbc_rating MODIFY points FLOAT NOT NULL;
-ALTER TABLE rbc_rating ALTER points DROP DEFAULT;
+ALTER TABLE rbc_rating ALTER points DEFAULT NULL;
```

Comment:
ALTER TABLE rbc_rating ALTER points DROP DEFAULT;
Is not a valid Oracle sentence.
Which is the intention?
If the intention is not to give a default value, the proposed statement is correct. Although I suspect that the default value should be 0 since previously those that had no value were set to 0

Reply:
This is case our database did have a default value for this column. I translated the MySQL code to this:

UPDATE rbc_rating SET points = 0 WHERE points IS NULL;
ALTER TABLE rbc_rating ADD points2 FLOAT;
UPDATE rbc_rating SET points2 = points;
ALTER TABLE rbc_rating DROP COLUMN points;
ALTER TABLE rbc_rating RENAME COLUMN points2 TO points;
ALTER TABLE rbc_rating MODIFY points FLOAT NOT NULL;

Reply:
Exactly @plukasew , that makes sense to me, I think we should do this.

Reply:
@vacercon WDYT?

Reply:
-- Esto no funciona
ALTER TABLE rbc_rating MODIFY points DROP DEFAULT; --ERROR DESA, TEST y PROD